packages/foam-vscode/src/services/editor.ts

```diff
@@ -1,6 +1,5 @@
 import { isEmpty } from 'lodash';
 import { asAbsoluteUri, URI } from '../core/model/uri';
-import { TextEncoder } from 'util';
```

Comment:
doesn't remove these import affect the native extension?

Reply:
Not required anymore, as it is default part from bode > 16.